src/reducers/reducerCreators.js

```diff
@@ -15,14 +15,17 @@ import { combineReducers } from 'redux';
 import keyBy from 'lodash.keyby';
 import merge from 'lodash.merge';
 
-import { typeToPlural } from '../utils';
+import { isStale, typeToPlural } from '../utils';
 
 function createByIdReducer({ type }) {
   const typePlural = typeToPlural(type);
   return function byId(state = {}, action) {
     switch (action.type) {
       case `${type}Created`:
       case `${type}Updated`:
+        if (isStale(action.payload, state)) {
```

Comment:
Anyway this could be a case rather than a condition in updated. Could we also have a race condition on created events ?

Reply:
yes both could have the same race condition, this already handles both as it falls through from the 'Created' case